tests/fixtures/explore.ts

```diff
@@ -44,7 +44,7 @@ export class ExplorePage {
   }
 
   async setDefaultViewportSize() {
-    await this.page.setViewportSize({ width: 1280, height: 720 });
+    await this.page.setViewportSize({ width: 1280, height: 680 });
```

Comment:
window height also changed a bit in 11.3, which had the consequence of adding more lazy loaded elements into the viewport, bumping down the height a tiny bit so we don't have to update a bunch of assertions.